src/components/Footer.js

```diff
@@ -10,10 +10,9 @@ import {
 
 const Footer = ({ snippets }) => {
   return (
-    <Segment
+    <Segment className="footer_segment"
```

Comment:
Just to let people know this is being discussed, we know this doesn't have to do with the redirect page and we're working out how to keep branches untangled.